mlforecast/core.py

```diff
@@ -245,10 +245,17 @@ def __init__(
     ):
         if isinstance(freq, str):
             self.freq = pd.tseries.frequencies.to_offset(freq)
+        elif isinstance(freq, pd._libs.tslibs.offsets.BaseOffset):
+            self.freq = freq
```

Comment:
we should document that `freq` can also be a pandas offset by adding it to the `Freq` type above and to the docstrings.

Reply:
fixed